python/private/py_executable.bzl

```diff
@@ -682,15 +682,30 @@ def _create_venv_symlinks(ctx, venv_dir_map):
 def _build_link_map(entries):
     # dict[str kind, dict[str rel_path, str link_to_path]]
     link_map = {}
+
+    # Here we store venv paths by package
```

Comment:
The type description makes it clear they're keyed by package, so no need to restate it in prose.

```suggestion
```